elements/lisk-p2p/src/p2p.ts

```diff
@@ -103,6 +103,7 @@ export const EVENT_FAILED_TO_ADD_INBOUND_PEER = 'failedToAddInboundPeer';
 export const EVENT_NEW_PEER = 'newPeer';
 
 export const NODE_HOST_IP = '0.0.0.0';
```

Comment:
This is redundant after introducing `NODE_HOST_IP_LIST.

Reply:
We are using `NODE_HOST_IP` only for http server that requires only one ip, if we remove it then it will look something like `this._httpServer.listen(this._nodeInfo.wsPort, NODE_HOST_IP_LIST[1])`
which doesn't look so nice. 

How about we rename them like below?,
`NODE_HOST_IP` => `NODE_HTTP_HOST_IP`
`NODE_HOST_IP_LIST` => `NODE_VALID_HOST_IP_LIST`

Reply:
removed this condition as discussed, so it isn't relevant anymore